corehq/apps/sms/urls.py

```diff
@@ -29,15 +29,13 @@
     upload_sms_translations,
 )
 from corehq.apps.smsbillables.dispatcher import SMSAdminInterfaceDispatcher
-from custom.icds.views.custom_sms_report import SMSUsageReport
 from corehq.messaging.smsbackends.telerivet.urls import \
     domain_specific as telerivet_urls
 
 urlpatterns = [
     url(r'^$', default, name='sms_default'),
     url(r'^send_to_recipients/$', send_to_recipients, name='send_to_recipients'),
     url(r'^compose/$', ComposeMessageView.as_view(), name=ComposeMessageView.urlname),
-    url(r'^message_report/$', SMSUsageReport.as_view(), name=SMSUsageReport.urlname),
```

Comment:
this changes the url from `https://www.icds-cas.gov.in/a/icds-cas/sms/message_report/` to `https://www.icds-cas.gov.in/a/icds-cas/custom_message_report/` but it does not impact the navigation or visibility of links etc.